pkg/kubectl/cmd/apiversions.go

```diff
@@ -51,19 +51,13 @@ func RunApiVersions(f *cmdutil.Factory, w io.Writer) error {
 		return err
 	}
 
-	apiVersions, err := client.ServerAPIVersions()
+	groupList, err := client.Discovery().ServerGroups()
 	if err != nil {
 		fmt.Printf("Couldn't get available api versions from server: %v\n", err)
```

Comment:
Actually, It seems like this should return the `err` to later be checked. Having the exit here makes it impossible to reasonable test this path.